src/datasources/userDataCreateUser.datasource.js

```diff
@@ -0,0 +1,18 @@
+import config from "../config";
+import request from "superagent";
+
+/**
+ * Create new user in userdata service
+ */
+export async function load({ smaugUserId }, context) {
+  const { url } = config.datasources.userdata;
+  const addUserEndpoint = url + "user/add";
+  await request.post(addUserEndpoint).send({ smaugUserId: smaugUserId });
```

Comment:
For at uddybe Pers kommentar: Fetches skal være noget i stil med:
```
const res = await context?.fetch(addUserEndpoint, {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
    },
    body: { smaugUserId: smaugUserId },
  });

  return res.body;
```

Reply:
done